src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Layout/NavMenu.razor

```diff
@@ -27,5 +27,37 @@
                 <span class="bi bi-list-nested" aria-hidden="true"></span> Weather
             </NavLink>
         </div>
+        @*#if (IndividualLocalAuth && UseWebAssembly)
+
+        <div class="nav-item px-3">
+            <NavLink class="nav-link" href="auth">
+                <span class="bi bi-lock" aria-hidden="true"></span> Auth Required
+            </NavLink>
+        </div>
+        ##endif*@
+        @*#if (IndividualLocalAuth)
+
+        <AuthorizeView>
+            <Authorized>
+                <div class="nav-item px-3">
+                    <NavLink class="nav-link" href="/Account/Manage">
+                        <span class="bi bi-person-fill" aria-hidden="true"></span> @context.User.Identity?.Name
+                    </NavLink>
+                </div>
+            </Authorized>
+            <NotAuthorized>
+                <div class="nav-item px-3">
+                    <NavLink class="nav-link" href="/Account/Register">
+                        <span class="bi bi-person" aria-hidden="true"></span> Register
+                    </NavLink>
+                </div>
+                <div class="nav-item px-3">
+                    <NavLink class="nav-link" href="/Account/Login">
+                        <span class="bi bi-person-badge" aria-hidden="true"></span> Login
+                    </NavLink>
+                </div>
```

Comment:
These are typically displayed on a separate menu option on the top right corner of the page as opposed to the general app navigation menu.

Reply:
The problem with using the top right corner for the identity navigation like we do for the other project templates and for the "About" link on this template, is that on this template, the top right navigation disappears for mobile and small screens.

![20230918-1945-29 6434625](https://github.com/dotnet/aspnetcore/assets/54385/e8870204-774f-4014-b65d-c21f97c88fb2)